framework/src/Volo.Abp.TextTemplating/Volo.Abp.TextTemplating.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Volo.Abp.TextTemplating.Scriban\Volo.Abp.TextTemplating.Scriban.csproj" />
+    <ProjectReference Include="..\Volo.Abp.TextTemplating.Razor\Volo.Abp.TextTemplating.Razor.csproj" />
```

Comment:
I think we should not add razor to this package. It will bring additional dependencies to current applications.
We keep this package for backward compatibility. We should mark it as obsolete by indicating to use other packages.